### DISCUSSION TITLE
Sidebar toggle not recognising the initial state

### DISCUSSION THREAD

**Discussion by mariomatasystems** [cables-gl/cables_docs#843] [! Welcome - readme first!]
Trying to style the toggle in the sidebar to include hover states.
Having this problem where the toggle isn't recognising the initial true state.

You'll see in this example that when you hover the toggle appears to switch.
After an initial interaction everything works as expected

https://dev.cables.gl/edit/AtVVVm

**Comment by pandrr** [comment]
fixed on dev.cables.gl let me know if i missed something
